src/index.js

```diff
@@ -2,31 +2,24 @@ const server = require('./server');
 const puppeteer = require('./puppeteer');
 const defaults = require('../config/defaults');
 
-module.exports = async function getRunner(
-  siteKey = defaults.SITE_KEY,
-  interval = defaults.INTERVAL,
-  port = defaults.PORT,
-  host = defaults.HOST
-) {
+module.exports = async function getRunner(siteKey, options = {port: defaults.PORT, host: defaults.HOST, interval: defaults.INTERVAL, threads: defaults.THREADS}) {
```

Comment:
This will fail when the user sends a partial options object. ie:

```js
function withDefaults(options = { host: 'localhost', port: 8080}){ 
  console.log(options.host + ':' + options.port);
}
withDefaults() // localhost:8080
withDefaults({ port: 8081 }) // undefined:8081
```

I think we should do something like this:

```js
const defaultOptions = {
  port: defaults.PORT, 
  host: defaults.HOST, 
  interval: defaults.INTERVAL, 
  threads: defaults.THREADS
};
module.exports = async function getRunner(siteKey, constructorOptions = defaultOptions) {
  const options = Object.assign({}, defaultOptions, constructorOptions);
  ...
```

Reply:
:+1: changed options keys